examples/expectations/column_aggregate_expectation_template.py

```diff
@@ -0,0 +1,390 @@
+import json
```

Comment:
What state is this file currently in?

If it doesn't work, we shouldn't merge it in this PR.

Reply:
This template works - no reason not to include it. Obviously, it can be improved, which we will do in the next few days.

Reply:
Awesome, I just wasn't sure about it's state.